### ISSUE TITLE
Table of Contents shown twice

### ISSUE THREAD

**Issue by desb42** [gnosygnu/xowa#283]
Continuing play with en.wikivoyage I have noticed another effect
![den_bosch_bad_toc](https://user-images.githubusercontent.com/861631/48690248-a8ed2a80-ebc5-11e8-9b96-ab8a11b62333.jpg)
wikivoyage produces a horizontal TOC (see blue oval) as part of Template:Pagebanner; however the TOC is also shown (see red oval)

A way to solve this is to use the TOC in the html only once.
Although I can see that there may be issues with the saved html version (hzip?)

**Comment by desb42** [comment]
due to my problems with #282, I have not actually generated any html databases. So the effect occurs due to the html being generated directly from wikitext and hence it looks like reason 2

**Comment by gnosygnu** [comment]
Makes sense. I've been pretty busy with work recently, and with Thanksgiving coming up, probably will not get a chance to look at it until the weekend.

**Comment by desb42** [comment]
Happy thanksgiving
